respy/parallelization.py

```diff
@@ -119,13 +120,18 @@ def split_and_combine_df(func=None, *, remove_type=False):
 
     def decorator_split_and_combine_df(func):
         @functools.wraps(func)
-        def wrapper_distribute_and_combine_df(df, *args, optim_paras, **kwargs):
-            dense_columns = create_dense_state_space_columns(optim_paras)
+        def wrapper_distribute_and_combine_df(
+            df, *args, optim_paras, period, dense_indexer, **kwargs
+        ):
+            dense_choice_columns = create_dense_state_space_columns(optim_paras)
```

Comment:
These are still dense columns, right?

Reply:
Or you called the wrong function.

Reply:
Or was this the circular import thingy? :)

Reply:
Still dense you are right! I did not delete the other thing that I have been thinking about but decided against after all